source/docs/software/troubleshooting/robots-shouldnt-quit.rst

```diff
@@ -0,0 +1,415 @@
+Robots Should Not Quit, But Yours Did!
+======================================
+
+``Robots should not quit, but yours did!``
+
+When your robot code hits an unexpected error, you will see this message show up in some console output (Driver Station or rioLog). You'll probably also notice your robot abruptly stop, or possibly never move. These unexpected errors are called *unhandled exceptions*.

```

Comment:
I don't think this capitalization of RioLog is correct.
![image](https://user-images.githubusercontent.com/13039555/116152343-ccc8ca80-a6b3-11eb-8013-4880d6909d05.png)
```suggestion
When your robot code hits an unexpected error, you will see this message show up in some console output (Driver Station or RioLog). You'll probably also notice your robot abruptly stop, or possibly never move. These unexpected errors are called *unhandled exceptions*.

```